modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/LinkOrButton.js

```diff
@@ -15,33 +15,55 @@
 import ClayButton from '@clayui/button';
 import ClayIcon from '@clayui/icon';
 import ClayLink from '@clayui/link';
-import React from 'react';
+import classNames from 'classnames';
+import React, {useContext} from 'react';
+
+import FeatureFlagContext from './FeatureFlagContext';
 
 const LinkOrButton = ({
```

Comment:
This is very convoluted, but we needed this component that already rendered a link or a button depending on the props it receives, to also display different contents depending on the screen size. 

I added the `responsive` variable that represents whether this component should render two different links/buttons: one with the icon, visible on small screens, and another with the text, visible on larger screens.

If the component receives both the icon and the text props, it will trigger responsive mode, rendering two different buttons/links that switch visibility depending on screen size.

![I hate myself](https://i.imgflip.com/3fr7ej.jpg)